src/web.ts

```diff
@@ -263,6 +263,20 @@ export async function app(probot: Probot, probotOptions: ApplicationFunctionOpti
       pendingPR.action = PRAction.Blocked;
       await db.setPendingPR(pendingPR);
 
+      // Append to description on PR to attach the issue to the pending PR
+      const closesMsg = `Closes #${issueID}`;
+      const appended = await appendPRDescription(
```

Comment:
I think a cleaner approach would have been to still throw an error if the API call fails but let the caller (the code here) choose to log and ignore it. The reason of the error is lost to the caller in this case.

Reply:
Good point--I'll make an update